website/docs/Examples/EMGProjects/EMG_Joystick.md

```diff
@@ -0,0 +1,68 @@
+---
+id: EMG_Joystick
+title: EMG Joystick
+---
+This tutorial will show you how to control a joystick using EMG data with the OpenBCI GUI. You can then use this customizable, muscle-activated joystick for any for any purpose of your choosing! We used this to control a drone as seen in the 2023 OpenBCI TED Talk.
```

Comment:
for any for any purpose <- duplicate words

OpenBCI used this to control a drone, as seen in the 2023 OpenBCI TED Talk.
^^^ Missing comma